docs/pywbemclicmdshelp.rst

```diff
@@ -925,18 +923,22 @@ The following defines the help output for the `pywbemcli connection select --hel
       connection. If the NAME argument is omitted, prompt for selecting one of
       the connection definitions in the connections file.
 
-      This command is useful in the interactive mode of pywbemcli.
+      Selects a connection from the persistently stored set of named connections
+      if NAME exists in the store. If NAME not supplied, a list of connections
+      from the connections definition file is presented with a prompt for the
+      user to select a connection.
 
-      The selection of the current connection is not saved across interactive
-      sessions of pywbemcli.
+      Selection is persistent; it is used as the server definition in commands
+      in the remainder of an interactive session and is saved in the connections
+      file and used as the current connection if there is no other server
+      definition (--server or --name or --mock-server).
```

Comment:
This is confusing. Does the command set the current connection (as it previously was described to do)? Or the default connection? Or both?